src/Benchmarks/Startup.cs

```diff
@@ -115,6 +115,11 @@ public void ConfigureServices(IServiceCollection services)
                 }
             }
 
+            if (Scenarios.Any("MemoryCache"))
```

Comment:
It's future-proofing in case we add more scenarios which start with "MemoryCache".